src/LinAlg/hiopMatrixRajaDense.cpp

```diff
@@ -1403,7 +1403,7 @@ void hiopMatrixRajaDense::row_max_abs_value(hiopVector &ret_vec)
     RAJA_LAMBDA(RAJA::Index_type i)
     {
       for(int j = 0; j < n_local; j++) {
-        double abs_val = fabs(Mview(i, j));
+        double abs_val = std::fabs(Mview(i, j));
```

Comment:
Same here: Use of `std::fabs` is questionable here because this code is supposed to be portable.